api/client/BUILD

```diff
@@ -1,16 +1,14 @@
 licenses(["notice"])  # Apache 2
 
-load(
-    "@envoy//bazel:envoy_build_system.bzl",
-    "envoy_package",
-)
-
-envoy_package()
-
-load("@envoy_api//bazel:api_build_system.bzl", "api_proto_library")
 load("@com_github_grpc_grpc//bazel:cc_grpc_library.bzl", "cc_grpc_library")
 
-api_proto_library(
+# TODO(oschaaf): `tools/check_format fix` insists on adding/changing
+# load("bazel:envoy_build_system.bzl", "envoy_package")
+# That breaks the build.
+# load("@envoy//bazel:envoy_build_system.bzl", "envoy_package")
```

Comment:
I think you can configure exclusions, see https://github.com/envoyproxy/envoy/blob/4825facf2451b234af737b5af8c792c31bca5d41/tools/check_format.py#L800. I think `check_format` should ideally have a finer grained notion, of saying "we want to do buildifier, but not the `envoy_package` jazz outside of the Envoy repo.

Reply:
Excluded at the file-level, added https://github.com/envoyproxy/nighthawk/issues/165 to track optimizing this